src/Nethermind/Nethermind.Consensus/Processing/BlockchainProcessor.cs

```diff
@@ -454,6 +456,11 @@ private void TraceFailingBranch(in ProcessingBranch processingBranch, Processing
             catch (InvalidBlockException ex)
             {
                 BlockTraceDumper.LogDiagnosticTrace(blockTracer, ex.InvalidBlock.Hash!, _logger);
+                Metrics.BadBlocks++;
+                if (Encoding.ASCII.GetString(ex.InvalidBlock.ExtraData).Contains(new BlocksConfig().ExtraData, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Instead of doing `new BlocksConfig().ExtraData` we could use variable or sth like that
`BlocksConfig.DefaultExtraData`